skopt/space/space.py

```diff
@@ -22,10 +22,13 @@ def __repr__(self):
 
 
 def check_dimension(dimension, transform=None):
-    """
+    """Turn a provided dimension description into a dimension object.
+
     Checks that the provided dimension falls into one of the
     supported types. For a list of supported types, look at
-    the documentation of `dimension` below.
+    the documentation of ``dimension`` below.
+
+    If ``dimension`` is already a ``Dimension`` instance, return it.
```

Comment:
Don't think we should (ab)use `check_dimension` for modifying a `Dimension` so added this sentence.